chord_metadata_service/chord/workflows/wdls/cbioportal_export.wdl

```diff
@@ -62,23 +57,17 @@ task katsu_dataset_export {
     # command block (tested with womtool-v78). Using triple angle braces made
     # interpolation more straightforward.
     command <<<
-        # Export results at export_path and returns http code 200 in case of success
-        RESPONSE=$(curl -X POST ~{true="" false="-k" validate_ssl} -s -w "%{http_code}" \
+        # Export results; Katsu returns a .tar.gz file
+        curl -X POST ~{true="" false="-k" validate_ssl} -s --fail-with-body \
```

Comment:
Nice, I didn't know about the --fail-with-body flag, probably useful for other workflows as well.